corehq/apps/sms/handlers/form_session.py

```diff
@@ -11,8 +12,10 @@
 from corehq.apps.sms.messages import *
 from corehq.apps.sms.util import format_message_list, get_date_format
 from corehq.apps.smsforms.app import _responses_to_text, get_responses
-from corehq.apps.smsforms.models import SQLXFormsSession
+from corehq.apps.smsforms.models import SQLXFormsSession, XFormsSessionSynchronization, \
+    get_channel_for_contact
 from corehq.apps.smsforms.util import critical_section_for_smsforms_sessions
+from dimagi.utils.logging import notify_error
 
 
 def form_session_handler(v, text, msg):
```

Comment:
Realize you didn't introduce this, but this could be a good time to rename `v` - `verified_number`?

Reply:
Ha, sure. Yeah I also always kind of disliked that. Um, maybe though I'd prefer to do that in a different PR just to keep this down to the minimal changes we need to make.